packages/ui/components/alert/Alert.tsx

```diff
@@ -19,6 +19,12 @@ export interface AlertProps {
 }
 export const Alert = forwardRef<HTMLDivElement, AlertProps>((props, ref) => {
   const { severity, iconClassName, CustomIcon, customIconColor } = props;
+  const [isVisible, setIsVisible] = useState(true);
+  const handleClose = () => {
+    setIsVisible(false);
+  };
+
+  if (!isVisible) return null;
```

Comment:
Nah, it's fine currently. We don't want to make the alert component closable on click. we have a toast component for that purpose.